src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -43,6 +46,19 @@ export class DictionaryModal extends React.Component {
       },
     });
   };
+
+  onChangeSupportedLocale = (arr) => {
```

Comment:
Hey @emmabaye , please name this function name the right way. Like a `do action` such as `changeSupportLocale`. `onChangeSupportLocale` is preferable for an event.

Reply:
Thanks for the feedback. However,  that is the function that is called when an `option` in the `select` component changes. It is called when an `onChange` event occurs as you can see [here on line 231](https://github.com/openmrs/openmrs-ocl-client/blob/42d13b140e5cdac842ff87de97f244f200392f30/src/components/dashboard/components/dictionary/common/DictionaryModal.jsx).

Reply:
I have made the changes.